autoapi/templates/python/function.rst

```diff
@@ -1,6 +1,6 @@
 {%- if obj.display %}
 
-.. function:: {{ obj.name }}({{ obj.args|join(',') }})
+.. function:: {{ obj.short_name }}({{ obj.args|join(',') }})
```

Comment:
That the nesting has changed or is not happening and therefore relative ref linking doesn't work. This is currently a blind spot in the tests.